CDCgov/prime-reportstream
# Updated schemaName to comply with new implementation

This PR updates the `schemaName`s in the flexion org settings to use the new required schema paths (#13530). This fixes [RS Orders and Results Broken](https://github.com/CDCgov/trusted-intermediary/issues/942) in the CDC Intermediary

Test Steps (local):
1. Run `./prime multiple-settings set --input ./settings/staging/0166-flexion-staging-results-handling.yml`
2. Send an ORU message to the waters api with the `flexion.simulated-lab` client
3. Verify the request went through without any errors and the HL7 <=> FHIR conversions work

## Checklist

### Testing
